docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -10,7 +10,12 @@ localization_priority: Priority
 The Outlook add-in API subset of the JavaScript API for Office includes objects, methods, properties, and events that you can use in an Outlook add-in.
 
 > [!NOTE]
-> This documentation is for a **preview** [requirement set](/office/dev/add-ins/reference/requirement-sets/outlook-api-requirement-sets). This requirement set is not fully implemented yet, and clients will not accurately report support for it. You should not specify this requirement set in your add-in manifest. Methods and properties that are introduced in this requirement set should be individually tested for availability before using them. You may also need to join the [Office Insider program](https://products.office.com/office-insider).
+> This documentation is for a **preview** [requirement set](/office/dev/add-ins/reference/requirement-sets/outlook-api-requirement-sets). This requirement set is not fully implemented yet, and clients will not accurately report support for it. You should not specify this requirement set in your add-in manifest. Methods and properties that are introduced in this requirement set should be individually tested for availability before using them.
```

Comment:
We use a standard include file for preview CDN/API referencing with Excel. Can that be used here too?

Reply:
will compare them...

Reply:
actually, which include file did you mean?

Reply:
This one: https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/includes/using-preview-apis.md

There's also an Excel-specific version that uses that include file as a base: https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/includes/using-excel-preview-apis.md

Reply:
I don't think this include is appropriate for this page. I patterned this note on the Excel one on [this page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-preview-apis). Perhaps this one (or some version) could become an include for the overall preview APIs, vs *this feature x*.

Reply:
Alright. It's something to consider going forward if we have to edit a bunch of preview things at once.